project2/src/components/Cart/CartItem/index.jsx

```diff
@@ -0,0 +1,89 @@
+import "./style.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMinus, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+import { Button } from "react-bootstrap";
+import { formatMoney } from "../../../utils";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { deleteProduct, updateQuantity } from "../../../store/Slice/cartSlice";
+import { useTranslation } from "react-i18next";
+import ModalComponent from "../../Modal";
+
+function CartItem(props) {
+  const { id, title, thumbnail, price, quantity } = props;
+  const [value, setValue] = useState(quantity);
+  const [show, setShow] = useState(false);
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+
+  const handleDeleteProduct = (id) => {
+    dispatch(deleteProduct(id));
+  };
+
+  const handleDecrement = (id) => {
+    setValue(parseInt(quantity) - 1);
```

Comment:
```suggestion
    setValue(prevState => parseInt(prevState) - 1);
```
Như này sẽ rõ ràng hơn nhé.
Ngoài ra thì có thể convert string => number bằng cách này cũng được `+prevState` `+quantity`